pkg/vpcmodel/nodesExplainability.go

```diff
@@ -195,7 +203,7 @@ func (c *VPCConfig) computeExplainRules(srcNodes, dstNodes []Node,
 
 // computeActualRules computes from the potentialRules the actualRules that actually enable traffic,
 // considering filtersExternal potential.filtersExternal (which was computed based on the RoutingResource)
-func (c *VPCConfig) computeRouterAndActualRules(details *rulesAndConnDetails) error {
+func (c *VPCConfig) computeActualRulesAndSpecifics(details *rulesAndConnDetails) error {
```

Comment:
1. Done
2. There are several (6) functions, Starting with `ExplainConnectivity` and called by it recursively, that needs `VPCConfig` and are defined as its method.  This function is one of these, it needs `VPCConfig` since it calls `c.getContainingConfigNode`. I agree that for some it seems to make more sense being `VPCConfig` than others; but better be consistent here.

Reply:
some "methods vs functions" concepts:
```
Methods are better used for:
    Operations that are intrinsically tied to the data of a specific type.
    Encapsulating behavior within a type, enhancing readability and maintainability.
    Object-oriented programming concepts in Go, such as defining behavior specific to a struct.
```
doesn't it make sense to change most of the VPCConfig methods in nodesExplainability to be functions rather than methods? 
they do not actually operate on VPCConfig objects, just use it as another argument..

Reply:
Not sure... lots of functions in other places especially `nodesConnectivity.go` are defined as member functions of `VPCConfig`; the logic here seems similar. In some cases, e.g. `func (c *VPCConfig) getFiltersEnablingRulesBetweenNodesPerDirectionAndLayer` there's a twin function in  `nodesConnectivity.go` .

Reply:
ok, will open a separate issue to revisit `VPCConfig` methods and consider consistent improvements for the entire pkg.